portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSummary.tsx

```diff
@@ -441,8 +438,6 @@ const mapState = (state: AppState) => ({
   ),
 });
 
-const connector = connect(mapState, {
-  setErrorSnackMessage,
-});
+const connector = connect(mapState, {});
```

Comment:
We can pass null instead of an empty object
```suggestion
const connector = connect(mapState, null);
```